ios/FastImage/FFFastImageView.m

```diff
@@ -18,16 +18,36 @@ - (id) init {
     self = [super init];
     self.resizeMode = RCTResizeModeCover;
     self.clipsToBounds = YES;
+    [self addObserver:self forKeyPath:@"currentLoopCount" options:NSKeyValueObservingOptionNew context:nil];
```

Comment:
Need to use KVO here to avoid adding code to the super `SDAnimatedImageView` class, which is in a separate package.